packages/upset/src/components/Root.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-shadow */
```

Comment:
What is shadowed here? Can we enable this rule again?

Reply:
I added this because the extProvenance initialization wasn't working when I changed the variable names. It's very possible that I just renamed the wrong variables.